Extensions/3D/Model3DRuntimeObject3DRenderer.ts

```diff
@@ -236,6 +236,31 @@ namespace gdjs {
       }
     }
 
+    _reloadModel(
+      runtimeObject: Model3DRuntimeObject,
+      instanceContainer: gdjs.RuntimeInstanceContainer,
+      objectData: Model3DObjectData
+    ) {
```

Comment:
`Model3DObjectData` should probably stays inside the object.
I guess you could add a `_reloadModel` on the object side that calls this method, a bit like it's done for `_updateModel`.

Reply:
I've changed it up a bit, I think its better now, although its now not safe to call the `_reloadModel` method on the renderer directly: it should be done on the runtime object instead, a comment warns against that.